package.json

```diff
@@ -32,11 +32,13 @@
     "express": "~4.9.6",
     "md5": "~2.0.0",
     "pg": "^4.4.3",
-    "sqlite3": "^3.1.0",
     "stats-lite": "~1.0.3",
     "util": "~0.10.3",
     "xtend": "~4.0.0"
   },
+  "optionalDependencies": {
```

Comment:
I don't think Minkelite should depend on sqlite3 _at all_. If its initialized and passed a sqlite3 db handle, it should use it, if its initialized with a pg db handle, it should use that.